Dockerfile

```diff
@@ -35,20 +35,22 @@ RUN cd src/ && \
 	git clone -b ${ROS_DISTRO}-devel https://github.com/kairproject/open_manipulator_msgs.git && \
 	git clone -b ${ROS_DISTRO}-devel https://github.com/kairproject/open_manipulator_simulations.git && \
 	git clone -b ${ROS_DISTRO}-devel https://github.com/kairproject/robotis_manipulator.git && \
-	git clone https://github.com/kairproject/kair_algorithms_draft.git
+	git clone https://github.com/gt-ros-pkg/hrl-kdl.git && \
+	git clone -b feat/docker https://github.com/kairproject/kair_algorithms_draft.git
```

Comment:
Will we be using `feat/docker` branch, or should this be on `master` branch?

Reply:
Updated, thanks.